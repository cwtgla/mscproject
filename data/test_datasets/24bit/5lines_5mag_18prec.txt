11.15115643
3.12009501
0.10238767
-5.98352113
-12.04582059