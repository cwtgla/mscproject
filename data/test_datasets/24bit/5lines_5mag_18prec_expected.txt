44
59
11
12
46
233
0
39
254
148
32
133
176
17
230